GDJS/Runtime/SceneLayer.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-2016 Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  /**
+   * Represents a layer of a container, used to display objects.
+   *
+   * Viewports and multiple cameras are not supported.
+   */
+  export class SceneLayer extends gdjs.Layer {
```

Comment:
Should this be a `RuntimeSceneLayer`? The naming of gdjs.Layer was a mistake to not have the "Runtime".